mod/ArchConsoleManager.cs

```diff
@@ -34,6 +36,8 @@ public class ArchConsoleManager : MonoBehaviour
         private List<float> gameplayConsoleTimers;
         private Material progressMat;
         private Text progressText;
+        private bool initialized = false;
+        private ArchipelagoSession session => Randomizer.APSession;
```

Comment:
This seems like it would break if the player switched profiles / changed connections, since IIUC we create ArchConsoleManager once near the top of Randomizer.Start() and never recreate it?

(still think the progress indicator might be better as a separate PR since the actual inventory stuff doesn't have this concern)

Reply:
It occurs to me that I probably don't _need_ to recreate the `APSession` object on every connection attempt. But that is how it currently works, and I'd definitely want to do manual tests to ensure that subscribing to MutliClient stuff during one connection still works after reconnections using that same session, before we rely on it for code like this. Which is probably not something we want to block this PR on.